nimble/controller/syscfg.yml

```diff
@@ -515,6 +515,14 @@ syscfg.defs:
         value: 0
         experimental: 1
 
+    BLE_LL_CHANNEL_SOUNDING:
+        description: >
+            Enable support for Channel Sounding feature.
+        restrictions:
+            - (BLE_VERSION >= 55) if 1
```

Comment:
draft spec says it's update for 5.4 so make it 5.4 here
if new spec is release with cs, we'll update to proper version